test/fixtures/remarkable/xss.txt

```diff
@@ -77,3 +77,12 @@ javascript:alert(1)
 <p>javascript:alert(1)</p>
 <p>javascript:alert(1)</p>
 .
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+src line: 82
```

Comment:
I've no idea what the number means, it feels like they used to have the actual line number at some point, but things have moved on?